overlord/configstate/configcore/vitality_test.go

```diff
@@ -164,6 +165,9 @@ func (s *vitalitySuite) TestConfigureVitalityWithQuotaGroup(c *C) {
 	r := systemd.MockSystemdVersion(248, nil)
 	defer r()
 
+	r = servicestate.EnsureQuotaUsability()
```

Comment:
interestingly enough, this test was working on accident, the `systemd.MockSystemdVersion` done above (and all the other ones in this package) were doing absolutely nothing to affect `systemdVersionError` in quota_control.go because it was run too late after the `init()` for servicestate was already executed. The reason the test was working without this sort of re-calculation of the error was because in this test (and in the other managers test that already was written), we sidestep some of the actual quota functions with `servicestatetest.MockQuotaInState`

Reply:
This is a really interesting find and nice catch!